package.json

```diff
@@ -191,6 +191,7 @@
     "howler": "^2.2.1",
     "js-yaml": "^4.1.0",
     "leaflet": "^1.4.0",
+    "linkedom": "^0.18.3",
```

Comment:
* [x] **Actively maintained based on recent commit history:** PRs merged regularly and recently. https://github.com/WebReflection/linkedom/pulls?q=is%3Apr+is%3Aclosed
* Code changes are secured by one of the following:
  * [x] **Only a small group of people have commit access:** Changes essentially only be one person https://github.com/WebReflection/linkedom/graphs/contributors
  * [ ] **Code review is enforced:** No. https://scorecard.dev/viewer/?uri=github.com/WebReflection/linkedom
* [x] **No binary artifacts:** None. https://scorecard.dev/viewer/?uri=github.com/WebReflection/linkedom
* [x] **Any unfixed vulnerabilities reported by GitHub are minor or not relevant:** None. https://scorecard.dev/viewer/?uri=github.com/WebReflection/linkedom
* [x] **Compatible license:** ISC https://github.com/WebReflection/linkedom/blob/main/LICENSE
* [x] **Reputable maintainer(s):** The repo owner and primary contributor is [WebReflection](https://github.com/WebReflection), whose GitHub profile is a member of the [NodeJS organization](https://github.com/nodejs). They were a [member of the Node.js modules team](https://github.com/nodejs/modules/blob/main/README.md), though that repo was archived in Sep 2023. They remain active in the Node.js community, as indicated by [opening issues recently](https://github.com/nodejs/node/issues/52173).
* [x] **Established project:** `WebReflection` has been working on the project since at least 2021: https://github.com/WebReflection/linkedom/pull/1

I think this dependency is okay to add. Leaving this comment to document my reasoning